logstash-core/api/config.ru

```diff
@@ -0,0 +1,22 @@
+ROOT = File.expand_path(File.dirname(__FILE__))
+$LOAD_PATH.unshift File.join(ROOT, 'lib')
+Dir.glob('lib/**').each{ |d| $LOAD_PATH.unshift(File.join(ROOT, d)) }
+
+require 'sinatra'
+require 'app/root'
+require 'app/pipeline'
+require 'app/system'
+
+env = ENV["RACK_ENV"].to_sym
```

Comment:
I don't think we need this, for us in reality the `RACK_ENV` is defined by the `--debug` option passed on the command line, no ?